packages/expo-store-review/android/src/main/java/expo/modules/storereview/StoreReviewModule.kt

```diff
@@ -53,7 +53,7 @@ class StoreReviewModule(private val mContext: Context) :
           }
         }
       } else {
-        promise.reject(null)
+        promise.reject("ERR_STORE_REVIEW_FAILED", "Android ReviewManager tasks was not successful")
```

Comment:
Error codes are documented in the [docs](https://docs.expo.dev/versions/latest/sdk/storereview/#error-codes) could you please update the docs to include this.

Reply:
Sure, I'll do this tomorrow. Thx for the review